platform/openide.awt/src/org/openide/awt/CloseButtonFactory.java

```diff
@@ -141,6 +141,9 @@ private static boolean isGTKLaF () {
     }
 
     private static Icon getCloseTabImage() {
+        if( null == closeTabImage ) {
+            closeTabImage = UIManager.getIcon("nb.close.tab.icon.enabled.icon"); //NOI18N
```

Comment:
Adding new keys to UIManager.getDefaults() constitutes an API change; should probably update apichanges.xml. And perhaps go over the names of the new keys one more time to make sure they follow the convention of the existing keys.

Reply:
While introducing new keys is an API change, the keys are not documented in ```apichange.xml```. That could be an additional task, or we could introduce a LAF guide on wiki. I would not hold this one off due to that.

Reply:
Sure--it's not worth any extra effort now.